test/test.hs

```diff
@@ -3888,7 +3911,8 @@ genEContract sz = do
   b <- defaultWord sz
   n <- arbitrary
   s <- genStorage sz
-  pure $ C c s b n
+  ts <- genStorage sz
+  pure $ C {code=c, storage=s, tStorage=ts, balance=b, nonce=n}
```

Comment:
can we rewrite this to:
```haskell
  code <- arbitrary
  balance <- defaultWord sz
  nonce <- arbitrary
  storage <- genStorage sz
  transientStorage <- genStorage sz
  pure C {code, storage, transientStorage, balance, nonce}
```

Reply:
Actually, I find this explicit initialization cleaner... What do you think @d-xo ? I wanted to write a version of yours but it didn't actually make the cut :D

Reply:
OK, so d-xo has no strong opinions, you are welcome to change it, I think it's fine, but please do push a commit if you feel about this more strongly :)